openpoiservice/utils/osm_reader.py

```diff
@@ -86,18 +86,20 @@ def area(self, obj_area):
             tags = self.extract_tags(obj_area.tags)
             # Determine centroid
             # TODO: import whole polygons, maybe simplified
-            wkb_area = wkb_fac.create_multipolygon(obj_area)
-            geom_multipolygon = wkblib.loads(wkb_area, hex=True)
-
-            for poly in geom_multipolygon:
-                self.store_objects(
-                    osmid=osmid,
-                    tags=tags,
-                    location=poly.centroid.wkt,
-                    osm_type=osm_type,
-                    categories=cat_included
-                )
-
+            try:
```

Comment:
It is because of inner polygon rings, see here: https://github.com/osmcode/libosmium/issues/225 
I will add this to the comment inline and add another issue.

Reply:
Actually, I won't add another issue, inner rings aren't needed, so the try/catch can stay there.

Reply:
Sorry wrong, we do want to handle polygons with inner rings but just use the outer. Will add an issue..

Reply:
In imposm world we did it this way: https://github.com/GIScience/openpoiservice/blob/master/openpoiservice/server/db_import/parse_osm.py#L112